nautobot/core/models/name_color_content_types.py

```diff
@@ -39,12 +39,14 @@ def get_for_models(self, models_):
 
 # TODO(timizuo): Inheriting from OrganizationalModel here causes partial import error
 class NameColorContentTypesModel(
-    BaseModel,
     ChangeLoggedModel,
+    ContactMixin,
     CustomFieldModel,
-    RelationshipModel,
-    NotesMixin,
     DynamicGroupMixin,
+    NotesMixin,
+    RelationshipModel,
+    StaticGroupMixin,
+    BaseModel,
```

Comment:
Would it be possible to move BaseModel on top so that the list is alpha sorted? or Does it cause a MRO issue?

Reply:
Caused an MRO issue. The mixins need to come before the base class.